MediaGalleryUi/Ui/Component/Listing/Columns/Layout/Options.php

```diff
@@ -0,0 +1,42 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Ui\Component\Listing\Columns\Layout;
+
+use Magento\Framework\Data\OptionSourceInterface;
+use Magento\Framework\Option\ArrayInterface;
+
+/**
+ * Image source filter options
+ */
+class Options implements ArrayInterface
+{
+    /**
+     * @inheritdoc
+     */
+    public function toOptionArray(): array
+    {
+        return [
```

Comment:
Can we inject this array here via di.xml? That will allow other extensions to easily add their entity to this.